### DISCUSSION TITLE
Tool to create gifs with PowerShell command execution for wiki content?

### DISCUSSION THREAD

**Discussion by FH-Inway** [d365collaborative/d365fo.tools#603] [Q&A] [answered]
Mostly a question for @Splaxi : Is there a tool to create those nice gifs that show the input and execution of the cmdlets in the wiki content? E.g. https://github.com/d365collaborative/d365fo.tools/wiki/How-To-Import-Bacpac-Into-Tier1

I would like to change one of them, because it uses a parameter where the name has changed (`-NewDatabaseName` of `Switch-D365ActiveDatabase` instead of `-SourceDatabaseName`).

**Comment by Splaxi** [comment, marked as answer]
ShareX is the tool used to capture the initial video / gif.

Screen2Gif is used to strip out all the 100% identical images in the gif, add custom wait time across all the images and duplicate the last image and set the wait for 1-3 seconds.

You should download one of gifs and open it with Screen2Gif to see what wait timeout is configured.
